src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -18,8 +18,10 @@ import browserSessionReplayPage from 'images/browser_screenshot-full_Session-rep
 
 import browserErrorsInboxSessionReplay from 'images/browser_screenshot-full_errors-inbox-session-replay.webp'
 
+import browserViewConsumptionDetails from 'images/browser_screenshot-crop_view-consumption-details.webp'
+
 <Callout variant="important">
-  Session replay is in limited preview. While not everyone can access it just yet, you can request access to the preview [here](https://newrelic.com/platform/session-replay-early-access).
+Session replay consumption will be billable starting May 8, 2024. See [Data consumption](#view-consumption) for details.
 </Callout>
 
 Session replay allows you to play back user interactions on your web app and map other telemetry data to help you understand your user's journeys. Session replay can be used for troubleshooting and improving the end-user experience, such as:
```

Comment:
Look for places where you can remove filler phrases and words like "allows you to". 

I'd replace it with something like "Session replay plays back user interactions ... and maps other telemetry data... etc" By removing those fillers, it often leads to stronger, more straightforward sentences.

Reply:
But otherwise, I this this is a very strong intro! I love the bullets with the concrete use cases and the way the animated GIF peeks up from below the fold. Well done!